SharedCode/build.gradle.kts

```diff
@@ -78,6 +78,9 @@ kotlin {
         implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime-common:$serialization")
         implementation("com.squareup.sqldelight:runtime:$sqlDelight")
         api("org.kodein.di:kodein-di:${Versions.kodein}")
+        api("org.reduxkotlin:redux-kotlin-threadsafe:0.5.1")
+        api("org.reduxkotlin:redux-kotlin-thunk:0.4.0")
+//        api("org.reduxkotlin:presenter-middleware:0.2.10")
```

Comment:
I couldn't get this dependency to resolve for the js client, but opened up an [issue](https://github.com/reduxkotlin/presenter-middleware/issues/1) on the repo to try and get this working.